utils.js

```diff
@@ -114,24 +116,29 @@ async function makeReactLayout() {
         } else if (node.type === 'comment') {
             return;
         } else if (node.type === 'text') {
-            const text = node.data;
+            const text = node.data; 
```

Comment:
Not a big deal here, and I'll update eslintrc to prevent end of line space, but in general watch out for PRs with unrelated changed lines.